config/locales/links/en.yml

```diff
@@ -1,7 +1,7 @@
 ---
 en:
   links:
-    access_help: Protect your data.
+    access_help: How login.gov protects your data.
```

Comment:
Should we also update the Spanish and French translations? In French, it would be `Comment login.gov protège vos données.`

Reply:
Spanish is probably `Cómo login.gov protege tus datos.` to keep the familiarity of the existing translation. Replace `tus` with `sus` to keep it in line with `links.two_factor_authentication.app_option`

Reply:
Sounds good. I went with `sus` to stay keep in line with other parts of the app. Thanks!